src/qt/transactionrecord.cpp

```diff
@@ -163,8 +164,31 @@ QList<TransactionRecord> TransactionRecord::decomposeTransaction(const CWallet *
                 if (ExtractDestination(txout.scriptPubKey, address))
                 {
                     // Sent to Bitcoin Address
-                    sub.type = TransactionRecord::SendToAddress;
-                    sub.address = CBitcoinAddress(address).ToString();
+										sub.address = CBitcoinAddress(address).ToString();
+										// Sent to testnet or mainnet vote / beacon address
+										if (sub.address == "mk1e432zWKH1MW57ragKywuXaWAtHy1AHZ" || sub.address == "S67nL4vELWwdDVzjgtEP4MxryarTZ9a8GB")
```

Comment:
this can be changed to `if (sub.address == GetBurnAddress())`

std::string GetBurnAddress() returns the correct burn address based on condition of it fTestNet is true or not